engine/config/locales/en.yml

```diff
@@ -45,9 +45,9 @@ en:
         last_update_html: Last updated <time id="page-updated-at" datetime="%{time}">%{time}</time>
         wording: Remember, you're doing a Good Job too!
       navbar:
-        cron_schedules: Cron Schedules
-        executions: All Executions
-        jobs: All Jobs
+        cron_schedules: Schedules
+        executions: Executions
+        jobs: Jobs
```

Comment:
I forgot to call this out in my initial summary. Are you ok with shortening these changes for the navbar?

Reply:
I think "Cron Schedules" should be shortened to "Cron". That's the keyword people are looking for.